packages/core/src/logger.ts

```diff
@@ -1,6 +1,28 @@
+import { setLogLevel as setClientSdkLogLevel } from 'livekit-client';
 import loglevel from 'loglevel';
 
-const log = loglevel.getLogger('lk-components-js');
+export const log = loglevel.getLogger('lk-components-js');
 log.setDefaultLevel('WARN');
 
-export default log;
+type LogLevel = Parameters<typeof setClientSdkLogLevel>[0];
+type SetLogLevelOptions = {
+  liveKitClientLogLevel?: LogLevel;
+};
+
+/**
+ * Set the log level for both the `@livekit/components-react` package and the `@livekit-client` package.
+ * To set the `@livekit-client` log independently, use the `liveKitClientLogLevel` prop on the `options` object.
+ * @public
+ */
+export function setLogLevel(
+  level: LogLevel,
+  options: SetLogLevelOptions = { liveKitClientLogLevel: undefined },
```

Comment:
nit: `liveKitClientLogLevel` could simply default to `level` instead of `undefined`, which makes the if clause redundant

Reply:
Good catch!